.github/workflows/build.yml

```diff
@@ -232,7 +232,8 @@ jobs:
         ../cpython-ro-srcdir/configure \
           --config-cache \
           --with-pydebug \
-          --with-openssl=$OPENSSL_DIR
+          --with-openssl=$OPENSSL_DIR \
+          --with-system-libmpdec=no
```

Comment:
I'm not wild about having to add this, since we're just going to have to remove it later.

Reply:
Ditto; I thought about adding a comment, but let's just record it as a follow-up item on the issue.

Reply:
Recorded on the issue; marking as resolved.

Reply:
I dug a little further on this, and it seems we can use the `libmpdec-dev` on Ubuntu 20.04 and 22.04. I'll update the PR to use this instead of modifying the CI.

Reply:
Addressed in a31cd57177ac88e15129264ebe0f9e552ddc82b2. Note that we need to upgrade the CI to Ubuntu 22.04 in order for this to fly:

- #118631

Reply:
CI was updated; marking this as resolved.